design-doc-template.adoc

```diff
@@ -112,6 +112,23 @@ or a confirmation that there are no security implications to consider.
 
 == Test Plan
 
+Depending on the selected stability level, the appropriate section below should be completed, including a brief description of how testing is to be performed in accordance with the selected stability level. The non-relevant sections may be removed as needed.
+////
+Depending on the stability level, the test plan required may vary. see below:
+////
+
+** Experimental - no test plan is required. Unit / integration tests may still be added as needed during development.
```

Comment:
s/may/should

Reply:
Perhaps

"Basic unit / integration tests should be added during development."

IOW people should write some tests but we're not going to be prescriptive about the details.